erpnext/www/support/index.html

```diff
@@ -0,0 +1,155 @@
+{% extends "templates/web.html" %}
+
+{% block content %}
+<section class="jumbotron bg-light">
+	<div class='container {{ align_greeting }}'>
+		<h1 class="h1">{{ _(greeting_text) }}</h1>
+		<p class="mt-4">
+			<div class="input-group shadow-sm {{ align_search_box }}">
+				<form action="/search_help" class="search-input">
+				<input name='q' class="form-control py-2 border-right-0 border" type="Text"
+				value='{{ frappe.form_dict.q or ''}}'
+				{% if not frappe.form_dict.q %}placeholder="{{ _("Search for help articles and more") }}"{% endif %}>
+				<span class="input-group-append">
+					<div class="border input-group-text bg-white"><i class="fa fa-search text-muted"></i></div>
+				</span>
+				</form>
+			</div>
+		</p>
+	</div>
+</section>
```

Comment:
```suggestion
<section class="section section-padding-top section-padding-bottom">
	<div class="container">
		<div class="hero-content">
			<h1>{{ greeting_text or _("We're here to help") }}</h1>
			<p class="hero-subtitle">{{ subtitle or _("Search for an answer or browse help topics.") }}</p>
		</div>
	</div>
</section>
```

* Use new section classes
* Remove search for now